extensions/amp-story/1.0/amp-story.js

```diff
@@ -1572,6 +1593,19 @@ export class AmpStory extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Block while waiting to resolve subscriptions page index.
+   * @param {string} targetPageId
+   * @param {!NavigationDirection} direction
+   * @return {!Promise}
+   * @private
+   */
+  blockOnPendingSubscriptionsPageIndex_(targetPageId, direction) {
+    return this.subscriptionsPageIndexPromise_.promise.then(() => {
+      return this.switchTo_(targetPageId, direction);
```

Comment:
You can use an arrow function without brackets if you're just returning inside it: `() => {return x;}` is the same as `() => x`